e2e/container-logs.e2e.spec.ts

```diff
@@ -0,0 +1,202 @@
+  });
+
+  test('should display container logs page', async () => {
+    const containerLogsPage = new ContainerLogsPage(page);
+
+    await containerLogsPage.waitForContainerInfo();
+    await expect(containerLogsPage.containerInfo).toBeVisible();
+
+    await containerLogsPage.waitForLogsToLoad();
+    await expect(containerLogsPage.terminal).toBeVisible();
+  });
+
+  test('should show container information', async () => {
+    const containerLogsPage = new ContainerLogsPage(page);
+
+    await expect(containerLogsPage.containerInfo).toBeVisible();
+
+    if (await containerLogsPage.containerName.count() > 0) {
+      const containerName = await containerLogsPage.getContainerName();
+      expect(containerName).toContain(testContainerName);
+    }
+
+    if (await containerLogsPage.containerState.count() > 0) {
+      const containerState = await containerLogsPage.getContainerState();
+      expect(containerState).toBeTruthy();
+    }
+  });
+
+  test('should display logs content', async () => {
+    const containerLogsPage = new ContainerLogsPage(page);
+
+    await containerLogsPage.waitForLogsToLoad();
+
+    const hasContent = await containerLogsPage.hasLogsContent();
+    expect(hasContent).toBe(true);
+
+    const terminalContent = await containerLogsPage.getTerminalContent();
+    expect(terminalContent).toContain('Hello from container logs');
+  });
+
+  test('should support log search', async () => {
+    const containerLogsPage = new ContainerLogsPage(page);
+
+    if (await containerLogsPage.searchInput.count() > 0) {
+      const searchTerm = 'Hello';
+      await containerLogsPage.searchLogs(searchTerm);
+
+      await page.waitForFunction(
+        () => {
+          const searchInput = document.querySelector('input[type="search"], input.search-input') as HTMLInputElement;
+          return searchInput && searchInput.value === 'Hello';
+        },
+        {timeout: 5000}
+      );
+
+      const inputValue = await containerLogsPage.searchInput.inputValue();
+      expect(inputValue).toBe(searchTerm);
+
+      if (await containerLogsPage.searchNextButton.count() > 0) {
+        await containerLogsPage.navigateSearchNext();
+      }
+
+      if (await containerLogsPage.searchPrevButton.count() > 0) {
+        await containerLogsPage.navigateSearchPrevious();
+      }
+
+      await containerLogsPage.clearSearch();
+
+      await page.waitForFunction(
+        () => {
+          const searchInput = document.querySelector('input[type="search"], input.search-input') as HTMLInputElement;
+          return searchInput && searchInput.value === '';
+        },
+        {timeout: 5000}
+      );
+
+      const clearedValue = await containerLogsPage.searchInput.inputValue();
+      expect(clearedValue).toBe('');
+    }
+  });
+
+  test('should handle terminal scrolling', async () => {
+    const containerLogsPage = new ContainerLogsPage(page);
+
+    await containerLogsPage.scrollToTop();
+    await containerLogsPage.scrollToBottom();
```

Comment:
I find that this doesn't actually work (that is, physically pressing <kbd>Home</kbd>/<kbd>End</kbd> does not actually move the scroll bar).  We should also ensure there _is_ scrolling and that the displayed text does update…